src/gpu-compute/GPUStaticInstFlags.py

```diff
@@ -108,4 +108,5 @@ class GPUStaticInstFlags(Enum):
         "MAC",  # MAC
         "MAD",  # MAD
         "MFMA",  # MFMA
+        "HAS_NO_ADDR",  # Request has no address but goes to SQC for timing
```

Comment:
Nit: This doesn't match TheStyle of the names above

Reply:
I had a problem where if it was `hasNoAddr` vs `HAS_NO_ADDR`, the compiler thought I was referring to the function `hasNoAddr`. I can try using the `::` scope to refer to the flag if I spell it camel case like you're suggesting and see if that works.

Reply:
I think @abmerop is suggesting HASNOADDR to be consistent with the other ones.  Are you saying that didn't work @mjkpolo ?

Reply:
oh sure I can do that instead, I think either `HasNoAddr` or `HASNOADDR` would work. @abmerop where you implying `HasNoAddr` by

> This doesn't match **TheStyle**

or `HASNOADDR`

Reply:
`NoAddr`